### PR TITLE
Ci fail

### PR DESCRIPTION
**Pull request by hector-chauran**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,5 +7,35 @@ jobs:
     steps:
```
Comment (hector-test-2) [inline .circleci/config.yml]: Test 1

File: .circleci/config.yml [hunk 2]
```diff
@@ -7,5 +7,35 @@ jobs:
...
       - run:
```
Comment (hector-test-2) [inline .circleci/config.yml]: Test 2

File: .circleci/config.yml [hunk 3]
```diff
@@ -7,5 +7,35 @@ jobs:
...
           name: Force failure
+          command: sleep 20 && exit 1
```
Comment (hector-test-2) [inline .circleci/config.yml]: Test 2

File: .circleci/config.yml [hunk 4]
```diff
@@ -7,5 +7,35 @@ jobs:
...
+
```
Comment (hector-test-2) [inline .circleci/config.yml]: Test 1

File: .circleci/config.yml [hunk 5]
```diff
@@ -7,5 +7,35 @@ jobs:
...
+  test:
+    docker:
```
Comment (hector-test-2) [inline .circleci/config.yml]: Test 2

File: .circleci/config.yml [hunk 6]
```diff
@@ -7,5 +7,35 @@ jobs:
...
+      - image: debian:stretch
+
```
Comment (hector-test-2) [inline .circleci/config.yml]: Test 3

File: .circleci/config.yml [hunk 7]
```diff
@@ -7,5 +7,35 @@ jobs:
...
+    steps:
```
Comment (hector-test-2) [inline .circleci/config.yml]: Test 1

File: .circleci/config.yml [hunk 8]
```diff
@@ -7,5 +7,35 @@ jobs:
...
+      - run:
```
Comment (hector-test-2) [inline .circleci/config.yml]: Test 2

File: .circleci/config.yml [hunk 9]
```diff
@@ -7,5 +7,35 @@ jobs:
...
+          name: Force success
```
Comment (hector-test-2) [inline .circleci/config.yml]: Test 3

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test 2

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test 3

File: .circleci/config.yml [hunk 10]
```diff
@@ -7,5 +7,35 @@ jobs:
...
           command: exit 1
- 
\ No newline at end of file
+
```
Comment (hector-test-2) [inline .circleci/config.yml]: Test 1

File: .circleci/config.yml [hunk 11]
```diff
@@ -7,5 +7,35 @@ jobs:
...
+  test2:
```
Comment (hector-test-2) [inline .circleci/config.yml]: Test 2

File: .circleci/config.yml [hunk 12]
```diff
@@ -7,5 +7,35 @@ jobs:
...
+    docker:
```
Comment (hector-test-2) [inline .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 10 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test 2

File: .circleci/config.yml (hunk 11 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test 3

File: .circleci/config.yml (hunk 10 above)
Comment (hector-test-2) [reply .circleci/config.yml]: Test 3

Comment (hector-chauran) [reply .circleci/config.yml]: Test 4

Comment (hector-test-2) [reply .circleci/config.yml]: Test 5

Comment (hector-chauran) [reply .circleci/config.yml]: Test 6

Comment (hector-chauran) [reply .circleci/config.yml]: Test 7

Comment (hector-chauran) [reply .circleci/config.yml]: Test 8

**Comment by hector-chauran** [comment]
Test

**Comment by hector-chauran** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-chauran** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

File: .circleci/config.yml (hunk 10 above)
Comment (hector-test-2) [reply .circleci/config.yml]: Test

**Comment by hector-chauran** [comment]
Test

**Comment by hector-chauran** [comment]
Test!

**Comment by hector-chauran** [comment]
@hector-test-2 Test

**Comment by hector-chauran** [comment]
Test

**Comment by hector-chauran** [comment]
@hector-test-2 Test 2

**Comment by hector-chauran** [comment]
@hector-test-2 Test

**Comment by hector-chauran** [comment]
@hector-test-2 Test 2

**Comment by hector-chauran** [comment]
@hector-test-2 Test

**Comment by hector-chauran** [comment]
@hector-test-2 Test

**Comment by hector-chauran** [comment]
@hector-test-2 Test

**Comment by hector-chauran** [comment]
@hector-test-2 Test

**Comment by hector-chauran** [comment]
@hector-test-2 Test

**Comment by hector-chauran** [comment]
@hector-test-2 Test

**Comment by hector-chauran** [comment]
@hector-test-2 Test!

**Comment by hector-chauran** [comment]
@hector-test-2 Test

**Comment by hector-chauran** [comment]
@hector-test-2 Test 50

**Comment by hector-chauran** [comment]
@hector-test-2 Test 505050

**Comment by hector-chauran** [comment]
Test!

**Comment by hector-chauran** [comment]
@hector-test-2 Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-test-2** [comment]
Test

**Comment by hector-chauran** [comment]
@hector-test-2 Test

**Comment by hector-test-2** [comment]
S

**Comment by hector-chauran** [comment]
@hector-test-2 Testing

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

**Comment by hector-chauran** [comment]
@hector-test-2 Test 505050

**Comment by hector-chauran** [comment]
@hector-test-2 Testing!

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Testing!

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Testing

Comment (hector-chauran) [reply .circleci/config.yml]: Testing

Comment (hector-chauran) [reply .circleci/config.yml]: Testing

Comment (hector-chauran) [reply .circleci/config.yml]: Testing

Comment (hector-chauran) [reply .circleci/config.yml]: Testing

Comment (hector-chauran) [reply .circleci/config.yml]: Testing

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Testing

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Testing

Comment (hector-chauran) [reply .circleci/config.yml]: Testing

Comment (hector-chauran) [reply .circleci/config.yml]: Testing

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 3 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 5 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 3 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 3 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 3 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 3 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 3 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 5 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 3 above)
Comment (hector-chauran) [reply .circleci/config.yml]: test

File: .circleci/config.yml (hunk 4 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 3 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 4 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 5 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 3 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 3 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 5 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 3 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 5 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 7 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Asd 12345

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 3 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 5 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 7 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml [hunk 13]
```diff
@@ -7,5 +7,35 @@ jobs:
     steps:
       - run:
           name: Force failure
```
Comment (hector-chauran) [inline .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 3 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 13 above)
Comment (hector-chauran) [inline .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 13 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 3 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 2 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

Comment (hector-chauran) [reply .circleci/config.yml]: Test

**Comment by hector-chauran** [comment]
Test

**Comment by hector-chauran** [comment]
@hector-test-2 Test

**Comment by hector-chauran** [comment]
@hector-test-2 Test

File: .circleci/config.yml (hunk 1 above)
Comment (hector-test-2) [reply .circleci/config.yml]: Test

**Comment by hector-chauran** [comment]
Test

**Comment by hector-chauran** [comment]
@hector-test-2 test

**Comment by hector-chauran** [comment]
@hector-test-2 Test

**Comment by hector-test-2** [comment]
.

**Comment by hector-chauran** [comment]
Test

**Comment by hector-chauran** [comment]
@hector-test-2 Test

**Comment by hector-chauran** [comment]
@hector-test-2 Testing

**Comment by hector-test-2** [comment]
@hector-chauran Test

**Comment by hector-chauran** [comment]
a

**Comment by hector-chauran** [comment]
a

**Comment by hector-chauran** [comment]
a

**Comment by hector-chauran** [comment]
a

**Comment by hector-chauran** [comment]
a

**Comment by hector-chauran** [comment]
a

**Comment by hector-chauran** [comment]
a

**Comment by hector-chauran** [comment]
a

**Comment by hector-chauran** [comment]
a

**Comment by hector-chauran** [comment]
a

**Comment by hector-chauran** [comment]
a

**Comment by hector-chauran** [comment]
a

**Comment by hector-chauran** [comment]
a

**Comment by hector-chauran** [comment]
Test message

**Comment by hector-chauran** [comment]
Test message

**Pull request closed without merging**